src/screens/Order/Orders.res

```diff
@@ -73,13 +84,77 @@ let make = (~previewOnly=false) => {
     }
   }
 
+  let updateViewsFilterValue = (view: TransactionViewTypes.viewTypes) => {
```

Comment:
As currentPage is already overloaded will move this logics inside TransactionViews